x/mongo/driver/topology/server.go

```diff
@@ -415,16 +448,16 @@ func (s *Server) ProcessError(err error, conn driver.Connection) driver.ProcessE
 
 		res := driver.ServerMarkedUnknown
 		// If the node is shutting down or is older than 4.2, we synchronously clear the pool
-		if cerr.NodeIsShuttingDown() || desc.WireVersion == nil || desc.WireVersion.Max < 8 {
+		if cerr.NodeIsShuttingDown() || wireVersion == nil || wireVersion.Max < 8 {
```

Comment:
What are your thoughts on creating a constant for "8"? Even though comparing wireVersions to their literal number is commonplace in the driver, it is a magic number which makes it sort of hard to interpret.

```go
const wireVersionGEQ42 = 8
```

Reply:
Yeah, using a named constant would help with readability. Will update.